physx/src/articulation_link.rs

```diff
@@ -115,53 +88,62 @@ pub trait ArticulationLink: Class<physx_sys::PxArticulationLink> + RigidBody + U
         ptr: *mut physx_sys::PxArticulationLink,
         user_data: Self::UserData,
     ) -> Option<Owner<Self>> {
-        Owner::from_raw((ptr as *mut Self).as_mut()?.init_user_data(user_data))
+        unsafe { Owner::from_raw((ptr as *mut Self).as_mut()?.init_user_data(user_data)) }
     }
 
     /// Get the user data.
+    #[inline]
     fn get_user_data(&self) -> &Self::UserData {
         // Safety: all construction goes through from_raw, which calls init_user_data
         unsafe { UserData::get_user_data(self) }
     }
 
     /// Get the user data.
+    #[inline]
     fn get_user_data_mut(&mut self) -> &mut Self::UserData {
         // Safety: all construction goes through from_raw, which calls init_user_data
         unsafe { UserData::get_user_data_mut(self) }
     }
 
     /// Enable collisions for this link. Equivalent to setting SimulationShape to false for all attached shapes.
+    #[inline]
     fn enable_collision(&mut self, enable: bool) {
         for shape in self.get_shapes_mut() {
             shape.set_flag(ShapeFlag::SimulationShape, enable);
         }
     }
 
-    /// Get inbound joint for this link
-    fn get_inbound_joint(&self) -> Option<&JointMap> {
-        unsafe {
-            (&PxArticulationLink_getInboundJoint(self.as_ptr())
-                as *const *mut physx_sys::PxArticulationJointBase as *const JointMap)
-                .as_ref()
-        }
-    }
+    // Get inbound joint for this link
+    // #[inline]
+    // fn get_inbound_joint(&self) -> Option<&JointMap> {
+    //     unsafe {
+    //         (&PxArticulationLink_getInboundJoint(self.as_ptr())
+    //             as *const *mut physx_sys::PxArticulationJointReducedCoordinate
+    //             as *const JointMap)
+    //             .as_ref()
+    //     }
+    // }
```

Comment:
Gone or just broken?

Reply:
The function is still there, but since I removed JointMap since all the base classes were removed I just commented this out for now, can add it back in a later version.